database/insertData/laos/03_template_license_MMC_MMD.js

```diff
@@ -436,7 +425,6 @@ exports.queries = [
                         title: "Education List"
                         elementTypePluginCode: "listBuilder"
                         category: QUESTION
-                        isEditable: false
```

Comment:
For all of the above. Isn't it misleading to allow the user to change this, but that doesn't update the record in the system? And if it does, I think the documents that prove need to be re-uploaded (this is regarding "Organisation category" basically)